.github/workflows/ci.yml

```diff
@@ -208,7 +208,7 @@ jobs:
           CORSO_M365_TEST_USER_ID: ${{ secrets.CORSO_M365_TEST_USER_ID }}
           CORSO_SECONDARY_M365_TEST_USER_ID: ${{ secrets.CORSO_SECONDARY_M365_TEST_USER_ID }}
           CORSO_PASSPHRASE: ${{ secrets.INTEGRATION_TEST_CORSO_PASSPHRASE }}
-          CORSO_LOG_FILE: stderr
```

Comment:
I'm not sure if this changes how the output is formatted in CI. I think we still log out everything if it fails. See line 221.

Reply:
Whoops, right you are. switched that to a file.